src/common/utility.h

```diff
@@ -121,11 +121,11 @@ OCSYNC_EXPORT Q_DECLARE_LOGGING_CATEGORY(lcUtility)
 #endif
 
     // convenience OS detection methods
-    inline bool isWindows();
-    inline bool isMac();
-    inline bool isUnix();
-    inline bool isLinux(); // use with care
-    inline bool isBSD(); // use with care, does not match OS X
+    constexpr inline bool isWindows();
```

Comment:
doesn't constexpr imply inline?

Reply:
Not necessarily. The function will be inlined as well, but `inline` also implies some definition rules according to the standard. I don't think it'll make a difference here. Nevertheless, as long as there is no reason to remove `inline`, I'd keep those specifiers.